### PR TITLE
point hds to master_bonnie now that bonnie-v3.1 was merged in that repo

### PR DESCRIPTION
**Pull request by lukoktonos**
Just updating the gemfile reference.  

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x JIRA ticket for this PR: N/A
- [x] JIRA ticket links to this PRN/A
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases N/A
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered. N/A
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass N/A

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @hackrm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases: N/A
- [x] You have tried to break the code: N/A

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass: N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree: N/A

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 gemspec
 
-gem 'health-data-standards', :git => 'https://github.com/projectcypress/health-data-standards.git', :branch => 'bonnie-v3.1'
+gem 'health-data-standards', :git => 'https://github.com/projectcypress/health-data-standards.git', :branch => 'master_bonnie'
```
Comment (jbradl11) [inline Gemfile]: I am thinking for HDS we should just publish the gem (tag the branch) and point to it in Bonnie_bundler / Bonnie now, since HDS contains such a minor change for this version, I don't see us needing to make a last minute change.

**Review by hackrm — APPROVED** [review]

**Review by hossenlopp — APPROVED** [review]
